index.html

```diff
@@ -1,14 +1,204 @@
+          color.
+          We’ll use Van Dyke Brown, Permanent Red, and a little bit of Prussian Blue. Mountains are so simple, they're
+          hard.
+          Let's put some happy little clouds in our world. And that's when it becomes fun - you don't have to spend your
+          time
+          thinking about what's happening - you just let it happen.
+
+          Let the paint work. Trees grow in all kinds of ways. They're not all perfectly straight. Not every limb is
+          perfect.
+          Isn't that fantastic that you can create an almighty tree that fast? How do you make a round circle with a
+          square
+          knife? That's your challenge for the day. Those great big fluffy clouds. Remember how free clouds are. They
+          just
+          lay
+          around in the sky all day long. As trees get older they lose their chlorophyll.
+          </br>
+          <a href="continue-reading.html"><button class="cr-button">Continue Reading</button></a>
+        </p>
+
+
+      </div>
+      <div class="container2">
+        <a href="blog-post.html"><button>July 30</button></a>
+        <h5>Just Another Post</h5>
+        <div class="border-below">
+          <p>This is the way you take out your flustrations. Maybe there's a happy little Evergreen that lives here. You
+            have to
+            make those little noises or it won't work.</p>
+        </div>
+        <a href="blog-post.html"><button>July 28</button></a>
+        <h5>And Another Post</h5>
+        <div class="border-below">
+          <p>You can create the world you want to see and be a part of. You have that power. Let's have a little bit of
+            fun
+            today. You're the greatest thing that has ever been or ever will be. You're special. You're so very special.
+          </p>
+        </div>
+        <a href="blog-post.html"><button>July 26</button></a>
+        <h5>One More Post</h5>
+        <div class="border-below">
+          <p>Everybody's different. Trees are different. Let them all be individuals. Water's like
+            me. It's laaazy ...
+            Boy,
+            it
+            always looks for the easiest way to do things</p>
+        </div>
+        <h5>Something of Note</h5>
+        <img class="small-image"
+          src="https://ih1.redbubble.net/image.1012577972.5741/st,small,507x507-pad,600x600,f8f8f8.jpg">
+        <div class="border-below">
+          <p class="float-text">We artists are a different breed of people. We're a happy bunch. If we're
+            gonna walk
+            though
+            the woods, we need a
+            little path. There isn't a rule. You just practice and find out which way works best for you. All those
+            little
+            son
+            of a guns. Follow the lay of the land. It's most important. Absolutely no pressure. You are just a whisper
+            floating
+            across a mountain.</p>
+          </br>

```

Comment:
Be careful about using your `</br>` tags for style purposes. In an instance like this, it's not the worst way to do it but adding padding or margin to the top of your button would be a better way to handle this type of styling.